samples/server/petstore/java-play-framework-api-package-override/app/Module.java

```diff
@@ -9,5 +10,6 @@ protected void configure() {
         bind(PetApiControllerImpInterface.class).to(PetApiControllerImp.class);
         bind(StoreApiControllerImpInterface.class).to(StoreApiControllerImp.class);
         bind(UserApiControllerImpInterface.class).to(UserApiControllerImp.class);
+		bind(SecurityAPIUtils.class);
```

Comment:
it is not generated properly, check here @bflamand

Reply:
yep, this was a tab vs space issue. Will be fixed in next push.